tests/util/cmd.go

```diff
@@ -80,6 +80,18 @@ func K3sCmd(cmdName string, cmdArgs ...string) (string, error) {
 	return string(byteOut), err
 }
 
+// K3sRemoveDataDir removes the provided directory as root
+func K3sRemoveDataDir(dataDir string) error {
```

Comment:
Are our integration tests ever run as a non-root users to warrant this?

Reply:
The problem is that the tmp dir created is 777, but then K3s fills all the subdirectories as root, so you can't call `os.RemoveAll` because it will fail with permissions errors.
I always run the integration tests (`go test ...) as a normal user, so does the GitHub CI.  I could strip out all the sudo stuff and assume that the  test will always be run as `sudo go test` or as root.

Reply:
Not needing sudo in the code would be preferred since that's user level thing.

Reply:
Addressing in another PR. Good for now.